specs/electra/weak-subjectivity.md

```diff
@@ -46,6 +46,30 @@ def compute_weak_subjectivity_period(state: BeaconState) -> uint64:
     return MIN_VALIDATOR_WITHDRAWABILITY_DELAY + epochs_for_validator_set_churn
 ```
 
+A brief reference for what these values look like in practice ([reference
+script](https://gist.github.com/jtraglia/af88887a0718b12f0543580dc195dd09)):
+
+| Safety Decay | Avg. Val. Balance (ETH) | Val. Count | Weak Sub. Period (Epochs) |
+| ---- | ---- | ---- | ---- |
+| 10 | 28000000000 | 32768 | 435 |
```

Comment:
What do you think about replacing `avg_balance * val_count` with total active balance, and then have a column with comment that this is N validators at 32 ETH balance? I think it is important to highlight that since Electra WS depends only on the total active balance. It also a bit confusing to have 36 ETH vals but not having 2048 ETH vals in the table

Reply:
Hmm yes, I can do this. Both you and Carl requested this change.